addon/components/polaris-drop-zone.js

```diff
@@ -559,5 +636,15 @@ export default Component.extend(ContextBoundEventListenersMixin, {
     this.setNode();
     this.set('state.error', this.get('error'));
     this.setupEvents();
+
+    if (this.get('openFileDialog')) {
+      this.triggerFileDialog();
+    }
+  },
+
+  didUpdateAttrs() {
+    if (this.get('openFileDialog')) {
```

Comment:
let's call `_super` here....you never know what we do in the future

Reply:
Ooh embarrassing 😳